plugins/wasi_http/func.cpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2024 Second State INC
+
+#include "func.h"
+#include "common/defines.h"
+#include "common/errcode.h"
+
+#include <cpr/cpr.h>
+#include <cstdint>
+#include <string>
+#include <vector>
+
+namespace WasmEdge {
+namespace Host {
+
+Expect<void> WasiHttpPrint::body(const Runtime::CallingFrame &, uint64_t Ptr,
+                                 uint64_t Len) {
+  std::string S{(const char *)Ptr, Len};
+  spdlog::info("print: {}", S);
```

Comment:
Add `[WASI-HTTP] ` prefix in the log statements.

Reply:
Don't forget to append `sv` to use string_literal.

Reply:
In this case we cannot use `sv`, spdlog will find no function matches.

Reply:
I missed `{}`. Got it.